lib/core/src/server/config/utils.js

```diff
@@ -29,8 +30,10 @@ export function loadEnv(options = {}) {
       env[name] = JSON.stringify(process.env[name]);
     });
 
+  const { stringified } = getEnvironment({ nodeEnv: JSON.parse(env.NODE_ENV) });
```

Comment:
I'm trying to put in an honest review, but I'm afraid you'll have way more context than me.

This line and 36 read as if you're specifically pulling out NODE_ENV and only applying that to environments. Is there anything to be worried about with other env vars not being passed, like `JEST_ENV`?

Reply:
Not all env vars get passed in Webpack. That was what the PR was about. Anything from the system, only `STORYBOOK_*` is allowed. You can add everything else to `.env` files.